### PR TITLE
added product page

### PR DESCRIPTION
**Pull request by Nagashree01**
added product page and filters by category, rating and price
productpage

Preview: https://deploy-preview-2--the-berry-store-5ed166.netlify.app/product.html

### CODE REVIEW

File: product.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    padding: 1rem;
+}
+.filter-name{
+    font-weight: 700;
+    padding: 0.8rem 0.2rem;
+}
+
+.product-list {
+    margin-left: 2rem;
+    padding-left: 20%;
```
Comment (shaksham08) [inline product.css:31]: You can also use shot hand syntax for padding here . Just a suggestion

**Review by shaksham08 — APPROVED** [review]

File: product.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    height: 100%;
+    width: 18%;
+    position: fixed;
+}
+
+.product-filters{
+    display:flex;
+    gap:2rem;
+    align-items: center;
+    padding: 0.3rem;
```
Comment (rushikesh3299) [inline product.css:17]: for resultant values less than 10px you can use px instead of rem. Rem is also ok.

**Review by rushikesh3299 — APPROVED** [review]
Overall looks good to me.

File: product.css (hunk 1 above)
Comment (Nagashree01) [reply product.css:31]: yes I'll do this

File: product.css (hunk 2 above)
Comment (Nagashree01) [reply product.css:17]: yes

**Pull request merged**